docs/network/bridge/README.md

```diff
@@ -0,0 +1,41 @@
+# Network Bridge Device Plugin
+
+This device plugin allows user to attach pods to bridges running on nodes.
```

Comment:
Aren't we assigning bridges to pods?

Reply:
Multiple pods can share one bridge. We attach pod (via veth pair) to a bridge.